sashimi-webapp/src/database/sql-related/sqlCommands.js

```diff
@@ -0,0 +1,261 @@
+import StringManipulator from '../stringManipulation';
+
+import exceptions from '../exceptions';
+
+import constants from '../constants';
+
+import DateTime from '../generated-data/dateTime';
+
+const alasql = require('alasql');
```

Comment:
Should keep the import syntax consistent

use `improt alasql from 'alasql';` instead